sdk/cosmosdb/cosmos/test/public/integration/fullTextSearch.spec.ts

```diff
@@ -9,7 +9,7 @@ import {
 } from "../common/TestHelpers.js";
 import { describe, it, assert, beforeAll } from "vitest";
 
-describe("Validate full text search queries", { timeout: 20000 }, () => {
+describe.skip("FTSQuery", { timeout: 20000 }, () => {
```

Comment:
let's keep the original name.

Reply:
Names without spaces actually help in running targeted test cases. Will improve the naming better when we unskip this test once emulator is updated.